From 117798f33463108d3f166c3c95c901deb0c0c835 Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 11 Jan 2024 09:33:25 +0100
Subject: [PATCH 0420/1695] plat-stm32mp2: panic of TZC interrupt

Enable CFG_STM32MP_PANIC_ON_TZC_PERM_VIOLATION on stm32mp2 platforms.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 34d7f97af9d0844e0bc928ab78d5e4d79c1f2df3 plat-stm32mp2: panic of TZC interrupt]
Change-Id: Ia8ff9c7f657ae9c1b7548a7e0a9b2b6112e3b73a
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 8f055b04f..4da601994 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -112,6 +112,9 @@ $(call force,CFG_WITH_SOFTWARE_PRNG,n,Mandated by CFG_HWRNG_PTA)
 CFG_HWRNG_QUALITY ?= 1024
 endif
 
+# Platform specific configuration
+CFG_STM32MP_PANIC_ON_TZC_PERM_VIOLATION ?= y
+
 # Enable reset control
 ifeq ($(CFG_STM32MP25_RSTCTRL),y)
 $(call force,CFG_DRIVERS_RSTCTRL,y)
-- 
2.47.2

